serving/docs/lmi/user_guides/vision_language_models.md

```diff
@@ -1,17 +1,16 @@
 # Vision Language Models in LMI
 
-> [!WARNING]
-> Vision Language Model support is currently experimental in v0.29.0.
-> LMI currently only supports using VLMs with the lmi-dist and vllm backends.
 
-Starting with v0.29.0, LMI supports deploying the following types of Vision Language Models:
+LMI supports deploying the following types of Vision Language Models:
```

Comment:
Should we specify about llama 3.2 with custom scripts support here?

Reply:
I think I'll add something for that here. I'm planning on making some demo notebooks, and then I'll link those here.